Makefile

```diff
@@ -441,3 +451,19 @@ cpplint:
 lint: jslint cpplint
 
 .PHONY: lint cpplint jslint bench clean docopen docclean doc dist distclean check uninstall install install-includes install-bin all staticlib dynamiclib test test-all test-addons build-addons website-upload pkg blog blogclean tar binary release-only bench-http-simple bench-idle bench-all bench bench-misc bench-array bench-buffer bench-net bench-http bench-fs bench-tls
+
+## Build data tarballs
+data-tarballs: $(LEDATATGZ) $(BEDATATGZ)
+
+$(LEDATATGZ) $(BEDATATGZ): node
+#	rm -rf $(BINARYNAME) out/deps out/Release
+#	$(PYTHON) ./configure --download=all --with-intl=full-icu
+	rm -rf $(TARNAME)
```

Comment:
I would not generate the localization data files in the same directory as where the source tarball's content is stored, just to avoid some interdependency between these two unrelated targets.